nilearn/decoding/space_net.py

```diff
@@ -18,6 +18,7 @@
 from functools import partial
 import numpy as np
 from scipy import stats, ndimage
+from sklearn.exceptions import NotFittedError
```

Comment:
Oops!, it didn't occur to me. I'll just issue a RuntimeError then.

On Sun, Nov 22, 2015 at 3:33 PM, Alexandre Abraham <notifications@github.com
> wrote:

> In nilearn/decoding/space_net.py
> <https://github.com/nilearn/nilearn/pull/851#discussion_r45559572>:
>
> > @@ -18,6 +18,7 @@
> >  from functools import partial
> >  import numpy as np
> >  from scipy import stats, ndimage
> > +from sklearn.exceptions import NotFittedError
>
> Is it compliant with the minimal version of sklearn requested? I think
> this error is something recent.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nilearn/nilearn/pull/851/files#r45559572>.
>



-- 
Elvis Dohmatob,
*sent from my  potato...*
http://dohmatob.blogspot.fr/, https://team.inria.fr/parietal/elvis/,
https://github.com/dohmatob